src/LinkDef.h

```diff
@@ -1,32 +1,19 @@
 #ifdef __MAKECINT__
-// #pragma link C++ defined_in "MagneticField/Engine/interface/MagneticField.h";
-// #pragma link C++ defined_in "MagneticField/Layers/interface/MagESector.h";
-// #pragma link C++ defined_in "MagneticField/Layers/src/MagBinFinders.h";
-// #pragma link C++ defined_in "MagneticField/Records/interface/IdealMagneticFieldRecord.h";
-// #pragma link C++ defined_in "MagneticField/VolumeGeometry/interface/MagVolume.h";
-// #pragma link C++ defined_in "MagneticField/VolumeBasedEngine/interface/VolumeBasedMagneticField.h";
-// #pragma link C++ defined_in "MagneticField/VolumeBasedEngine/interface/MagGeometry.h";
-// #pragma link C++ defined_in "FWCore/Framework/interface/EventSetup.h";
-// #pragma link C++ defined_in "FWCore/Framework/interface/EventSetupRecord.h";
-// #pragma link C++ defined_in "FWCore/Framework/interface/EventSetupRecordImplementation.h";
-// #pragma link C++ defined_in "DataFormats/FWLite/interface/ESHandle.h";
-// #pragma link C++ defined_in "MagneticField/UniformEngine/interface/UniformMagneticField.h";
-// #pragma link C++ defined_in "MagneticField/VolumeGeometry/interface/VolumeSide.h";
-// #pragma link C++ defined_in "DataFormats/GeometrySurface/interface/Surface.h";
-// #pragma link C++ class std::atomic<char>+;
-// #pragma link C++ class __atomic_base<char>+;
-// #pragma link C++ class __atomic_base<const MagVolume*>+;
-// #pragma link C++ class VolumeBasedMagneticField+;
-
-
-
-
-
-
-
-
-
-
+#pragma link C++ defined_in "MagneticField/Engine/interface/MagneticField.h";
+#pragma link C++ defined_in "MagneticField/Layers/interface/MagESector.h";
+#pragma link C++ defined_in "MagneticField/Layers/src/MagBinFinders.h";
+#pragma link C++ defined_in "MagneticField/Records/interface/IdealMagneticFieldRecord.h";
+#pragma link C++ defined_in "MagneticField/VolumeGeometry/interface/MagVolume.h";
+#pragma link C++ defined_in "MagneticField/VolumeBasedEngine/interface/VolumeBasedMagneticField.h";
+#pragma link C++ defined_in "MagneticField/VolumeBasedEngine/interface/MagGeometry.h";
+#pragma link C++ defined_in "FWCore/Framework/interface/EventSetup.h";
+#pragma link C++ defined_in "FWCore/Framework/interface/EventSetupRecord.h";
+#pragma link C++ defined_in "FWCore/Framework/interface/EventSetupRecordImplementation.h";
+#pragma link C++ defined_in "DataFormats/FWLite/interface/ESHandle.h";
+#pragma link C++ defined_in "MagneticField/UniformEngine/interface/UniformMagneticField.h";
+#pragma link C++ class std::atomic<char>+;
+#pragma link C++ class __atomic_base<char>+;
+#pragma link C++ class __atomic_base<const MagVolume*>+;
```

Comment:
revert this back to my commented version. 
Again, this needs to be moved to a different branch

Reply:
done, it's all commented